docs/source/basics/overview.rst

```diff
@@ -27,12 +27,48 @@ The Morpheus CLI is built on the Click Python package which allows for nested co
 together. At a high level, the CLI is broken up into two main sections:
 
  * ``run``
-    * For running NLP or FIL pipelines.
+    * For running AE, FIL, NLP or OTHER pipelines.
  * ``tools``
-    * Tools/Utilities to help set up, configure and run pipelines and external resources
+    * Tools/Utilities to help setup, configure and run pipelines and external resources.
```

Comment:
It's my understanding that when its a verb its two words "set up", but the noun is "setup".

I think this one is a verb.

Reply:
You're right. good catch.